docs/index.html

```diff
@@ -106,6 +106,16 @@
   };
   </script>
   <script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>
+  <script>
+    function closechart(params) {
+      const buts = document.getElementsByClassName("gitter-chat-embed-action-bar-item");
```

Comment:
What does `buts` stand for? We should prefer meaningful code that is easy to understand.

Reply:
I think it is for button ?

may be it should be rename to `btn` instead or something else, `buts` doesn't sound good 👍